src/code/z_bgcheck.c

```diff
@@ -527,10 +560,10 @@ void StaticLookup_AddPoly(StaticLookup* lookup, CollisionContext* colCtx, Collis
  * Locates the closest static poly directly underneath `pos`, starting at list `ssList`
  * returns yIntersect of the closest poly, or `yIntersectMin`
  * stores the pointer of the closest poly to `outPoly`
- * if (flags & 1), ignore polys with a normal.y < 0 (from vertical walls to ceilings)
+ * if BGCHECK_GROUND_CHECK_ON, ignore polys with a normal.y < 0 (from vertical walls to ceilings)
```

Comment:
```suggestion
 * if groundChk & BGCHECK_GROUND_CHECK_ON, ignore polys with a normal.y < 0 (from vertical walls to ceilings)
```

Reply:
feels unnecessary since `BGCHECK_GROUND_CHECK_ON` on its own expresses itself as being a groundChk flag in the name

Reply:
Well the function has a gazillion arguments

Reply:
I dont think you necessarily need to show the anding operation in the comment, but it can say something like "if `BGCHECK_GROUND_CHECK_ON` is passed" or "if `BGCHECK_GROUND_CHECK_ON` is set" to make it read more like a sentence 

but yea I do think it should be clear just by association of the name what its referring to.
But in that same vein, `groundChk` should just be `groundCheck` with the whole word, its only two characters and it will better match the define names

Reply:
it isn't 2 characters, it's 20, and it would introduce an inconsistent variable names with some abbreviated some not. Unless perhaps you want to eliminate the `chk` abbr altogether which in that case it would be about 262 characters in bgcheck alone.

Reply:
obviously i meant 2 characters per usage of the word... and I dont think the total amount of characters it adds is really relevant, when all that really matters is how long individual lines get

its already inconsistent with the enum name, so pick your posion. I think the abbreviating is not necessary and not helping to solve any particular problem, it would be fine as the whole word

Reply:
The bigger picture point for me and this PR is that `chk` is an established abbreviation in the code base at the moment, primarily in bgcheck, camera, Math3D, and inherited from event_chk_inf. If groundChk should just be groundCheck, then arguably all chks should be checks, but I'd rather avoid doing something like that in this PR since it falls outside of the scope of what I want to do with this PR. 

Either way, `groundChk` and `GROUND_CHECK` are equivalent to myself at least.